test/test_other.py

```diff
@@ -2742,10 +2742,10 @@ def test_embind(self, extra_args):
       (['-lembind', '-O1']),
       (['-lembind', '-O2']),
       (['-lembind', '-O2', '-sALLOW_MEMORY_GROWTH', test_file('embind/isMemoryGrowthEnabled=true.cpp')]),
+      (['-lembind', '-O2', '--closure=1']),
     ]
+    extra_args = extra_args + ['-sRETAIN_COMPILER_SETTINGS', '-sEXPORTED_RUNTIME_METHODS=getCompilerSetting']
     test_cases = [t + extra_args for t in test_cases]
-    # closure compiler doesn't work with DYNAMIC_EXECUTION=0
-    test_cases.append((['-lembind', '-O2', '--closure=1']))
```

Comment:
Did something change here?

Reply:
Also this file doesn't look like it contains my latest changes from #18862.

Reply:
This was moved up. Without moving it up it will not get the changes from line 2747 which add two flags to all the above test modes.

Reply:
(also merged in main now)

Reply:
So I think you can now also completely remove `helpers.js`

Reply:
Is the comment about `DYNAMIC_EXECUTION` no longer true then?

Reply:
Yes, this comment just stale and should have been removed before.